ufl/finiteelement/finiteelement.py

```diff
@@ -236,3 +237,9 @@ def __getnewargs__(self):
                 None,
                 self.quadrature_scheme(),
                 self.variant())
+
+    def is_fully_continuous(self):
+        """Return true if the values of this element's basis functions are continuous between elements."""
+        if self.family() == "Real":
+            return True
+        return self._sobolev_space in [H1, H2]
```

Comment:
The Sobolev spaces support `in` for elements and also know about which spaces are subspaces of each other. I therefore think `self in H1` would work here.